test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -513,6 +513,13 @@ def forward(self, x):
             # 1. Pair of to_int8 and to_fp32 at conv input * 1, extra input of add * 1, and graph output * 1
             #    matched in pointless_convert pass at
             #    torch/_inductor/fx_passes/joint_graph.py: [convert_element_type, convert_element_type_1]
+            #    NB: since quant workflow now duplicates DQ node, for each user, we wont necessarily see
```

Comment:
Hi @kimishpatel, could you also update the comments here https://github.com/leslie-fang-intel/pytorch/blob/59e5ce92f8b5ca68205df0abc444d0b55f16fc58/test/inductor/test_mkldnn_pattern_matcher.py#L514-L515 and other similar places? so, it will be aligned with expected count values.